cip.go

```diff
@@ -345,6 +345,12 @@ func main() {
 		os.Exit(0)
 	}
 
+	// Check the pull request
```

Comment:
This check can't run unconditionally under main(). It should only run when we are doing a PR on dry run when comparing the PR vs master. You should use the PULL_PULL_REF and other env vars to determine whether you are running in a PR or not. Or just pass an explicit flag in.